main/test/ca/uwaterloo/flix/api/lsp/TestCompletionProvider.scala

```diff
@@ -131,140 +132,159 @@ class TestCompletionProvider extends AnyFunSuite {
     */
   private val Limit: Int = 100
 
-  test("No crashes when calling getCompletions anywhere") {
-    Programs.foreach(program => {
-      val (root, errors) = compile(program)
-      program.scanLeft(Position(1, 1))({
-        case (Position(line, _), '\n') => Position(line + 1, 1)
-        case (Position(line, col), _) => Position(line, col + 1)
-      }).foreach(pos => CompletionProvider.getCompletions(Uri, pos, errors)(root, Flix).map(_.toCompletionItem(Flix)))
-    })
+  /////////////////////////////////////////////////////////////////////////////
+  // General Properties
+  /////////////////////////////////////////////////////////////////////////////
+
+  test("Autocomplete is well-defined") {
```

Comment:
Should `test` be invoked within the loop so that each is considered a separate test?